.travis.yml

```diff
@@ -39,8 +38,13 @@ matrix:
       env: SCRIPT=windows-nsis/build-snapshot
     - compiler: ": complete"
       env: SCRIPT=windows-nsis/build-complete
+    - compiler: ": snapshot"
+      env: SCRIPT=windows-nsis/build-snapshot OPENSSL_VERSION=1.1.0g
   exclude:
     - compiler: gcc
+  allow_failures:
+    - compiler: ": snapshot"
+      env: SCRIPT=windows-nsis/build-snapshot OPENSSL_VERSION=1.1.0g
```

Comment:
Is there a particular reason to repeat an identical "env:" line here?

Reply:
it is how "allow_failures" works.

currently, even snapshot build fail on openssl-1.1.0g, so we (temporary) mark them as allowed failures.

Reply:
Ok. Quick search [confirmed this](https://stackoverflow.com/questions/20905969/travisci-how-to-allow-failures-for-environment-variable#20909996).